parsec/api/protocol/organization.py

```diff
@@ -52,6 +53,7 @@ class OrganizationBootstrapWebhookSchema(BaseSchema):
     device_label = fields.String(allow_none=True, required=True)
     human_email = fields.String(allow_none=True, required=True)
     human_label = fields.String(allow_none=True, required=True)
+    active_users_limit = fields.Integer(allow_none=True, required=True)
```

Comment:
following https://github.com/Scille/parsec-cloud/pull/1766#discussion_r657746730

> The expiration date is always set by the SaaS instead of the user's limit that is set by the backend. The SaaS need to know the limit to send its mails.

The default user limit is configured in the backend once, so the SAAS should know about it.

I'm reluctant to add this field given it would mean we should also add `user_profile_outsider_allowed` (given it is also a per-organization configuration set by the backend) and later on all the other configuration needed...

I think it would be simpler to consider the default values known by the SAAS (of course this mean if the default values evolve, the SAAS should be re-deployed in sync with the backend) or just to have the SAAS send an `organization_status` command to the backend every time it receive a webhook (the only complexity here is to make sure we handle errors such as backend not available by logging the content of the webhook so we can manually retry later)

Reply:
@touilleMan Ok, I will set the default value in the SAAS.